src/components/organisms/WorldAdvancedForm/WorldAdvancedForm.tsx

```diff
@@ -60,8 +60,10 @@ export const WorldAdvancedForm: React.FC<WorldAdvancedFormProps> = ({
     () => ({
       attendeesTitle: world.attendeesTitle,
       chatTitle: world.chatTitle,
-      showNametags: world.showNametags,
+      radioStation: world.radioStations?.[0],
```

Comment:
I thought you have changed the typings and you should have `world.radioStation` here? Also, why do we remove showNametags here?

Reply:
1. I don't know the reasoning behind the addition of an array in the first place and some legacy, at least on staging, have/may have an array of stations, so leaving it for some next PR to determine if there is a desire for a list of stations to be used (even if just for a drop down suggestion in an input, not necessarily playing them one after another)

2. Not removed, just moved
![sparkle-nametags-01](https://user-images.githubusercontent.com/79229621/140741686-e34750eb-acc0-4fd8-a98e-12f15dce6258.png)